### DISCUSSION TITLE
no matches found: python-telegram-bot[job-queue]

### DISCUSSION THREAD

**Discussion by sournk** [python-telegram-bot/python-telegram-bot#3779] [Q&A] [answered]
Could you help me to install job-queue extension in my venv, please. 
I get an error `no matches found: python-telegram-bot[job-queue]` when I run `pip install python-telegram-bot[job-queue]`. What is the problem?

Python 3.9.13
pip 23.1.2

**Comment by Bibo-Joshi** [comment, marked as answer]
Try `pip install "python-telegram-bot[job-queue]"`

**Reply by sournk** [reply]
Thank you. It works for me.
